feature-libs/user/profile/styles/_new-password.scss

```diff
@@ -0,0 +1,43 @@
+%cx-new-password {
+  .myaccount-password-header {
+    font-size: $h3-font-size;
+    font-weight: var(--cx-font-weight-semi);
+    color: $text;
+    padding-bottom: 1.5rem;
+  }
+
+  .myaccount-password-button {
+    font-weight: var(--cx-font-weight-normal);
+  }
+
+  .myaccount-password-button-cancel {
+    color: $primary;
+  }
+
+  .myaccount-password-label {
+    font-weight: var(--cx-font-weight-semi);
+    min-width: 7.5rem;
+    padding-right: 1.2rem;
+    color: $background-dark;
+  }
+
+  .form-control::placeholder {
+    color: $medium;
+  }
+
+  .myaccount-label-padding {
+    padding-bottom: 0.75rem !important;
```

Comment:
Maybe we need remove important?

Reply:
It won't take effect if removing important!